app/src/test/java/org/feature/fox/coffee_counter/di/services/network/ApiServiceTest.kt

```diff
@@ -344,7 +346,7 @@ class ApiServiceTest {
 
         mockWebServer.enqueue(response)
 
-        val actualResponse = apiService.deleteUser(userId)
+        val actualResponse = apiService.deleteUser(bearerToken, userId)
```

Comment:
The test cases should also assert if the authorization header was send correctly to mock webserver